lib/index.js

```diff
@@ -40,29 +41,39 @@
   };
 
   const selfTests = {
-    arango: function (args) {
+    arango: function (args, options) {
       if (arango === undefined) {
         fatal("arango object is undefined. This should not happen. " +
               "Please try invoking this script via the arangosh!");
       }
+      if (arango.isConnected()) {
```

Comment:
Should we throw an error if not connected?

Reply:
analyze works on both an agency as well as dump